datasets/tess.yaml

```diff
@@ -0,0 +1,24 @@
+Name: Transiting Exoplanet Survey Satellite (TESS)
+Description: |
+  The Transiting Exoplanet Survey Satellite (TESS) is a two-year survey that will discover exoplanets in orbit around bright stars.
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Monthly
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a 'Requester Pays' bucket
+    ARN: arn:aws:s3:::stpubdata
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: Notifications for new data
+    ARN: arn:aws:sns:us-east-1:879230861493:stpubdata
+    Region: us-east-1
+    Type: SNS Topic
+DataAtWork:
```

Comment:
Would it be useful to add these links here too? 

- https://archive.stsci.edu/tess/
- https://heasarc.gsfc.nasa.gov/docs/tess/

Reply:
If the links are specifically talking about how to use the data on AWS, then I think they're ok to include in `DataAtWork`. I don't think that's the case though, so I would recommend calling them out in the `Description` section. You can use markdown to make the links look nice.